bundles/org.openhab.binding.ws980wifi/src/main/resources/OH-INF/i18n/ws980wifi_xx.properties

```diff
@@ -0,0 +1,21 @@
+# FIXME: please substitute the xx with a proper locale, ie. de
```

Comment:
Please remove this file.

Reply:
Generate the `en` version using `mvn i18n:generate-default-translations`

Reply:
deleted
running the suggested command from Hilbrand causes the following ERROR:
No plugin found for prefix 'i18n' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/er/.m2/repository), central (https://repo1.maven.org/maven2), openhab-release (https://openhab.jfrog.io/openhab/libs-release)]

Reply:
Might be necessary to run from root of repo, and add ` -pl :org.openhab.binding.ws980wifi` to the mvn command.